### PR TITLE
dt-bindings: pinctrl: renesas: Update r8a77951 & r8a779f0 pin list

### PR DESCRIPTION
**Pull request by aaillet**
Current pins and pin functions list were generated at each soc support creation.
It appeared that used script would not have filled in these lists perfectly.
Complete these lists based on socs documentation.

### CODE REVIEW

**Review by gmarull — APPROVED** [review]

File: include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h
Comment (xakep-amatop) [inline include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h:90]: I see no reason for these changes, and they create new conflicts with an already existing PR: https://github.com/zephyrproject-rtos/zephyr/pull/60285.

Could you please wait until the mentioned PR is merged and make these changes in all the necessary places, or alternatively, abandon these modifications?

**Review by xakep-amatop — CHANGES_REQUESTED** [review]

File: include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h
Comment (aaillet) [reply include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h:90]: Hmm, sorry but I can't see the conflict, #60285 is not editing file under `include/zephyr/dt-bindings/pinctrl/renesas`.
Am I wrong ?

I was just taking advantage of this PR to harmonize the file, but they can be removed.

**Comment by aaillet** [comment]
> something wrong with commit message:
> 
> ```
> List generated at r8a779f0 soc support creation was not complete.
> Complete it based on r8a77951 doc.
> ```

Thanks, fixed !

File: include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h
Comment (xakep-amatop) [reply include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h:90]: > Am I wrong ?

No, of course. I'm just trying to say that after your changes, I'll have a little bit more pain in my life :face_exhaling:, because it will be needed to update my PR in which I've [referenced the names that you changed](https://github.com/zephyrproject-rtos/zephyr/pull/60285/files#diff-4f51a3fd64a2a3c77aa6b5c17cc4d1d11d7548095c2a3a12d68d3be1a933d762R27). It will increase the review even more because, after the update, I should gather ack(s) again, and experience says that it takes at least 2 weeks.

Of course, I'm not able to mark this PR with NW because of this reason. This is just a gentle request.

**Review by lorc — APPROVED** [review]

File: include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h
Comment (aaillet) [reply include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h:90]: Sorry, I hadn't noticed you used this pin in the board pinctrl file.
I can add DNM to my PR waiting for your's (that is indeed harder to merge) to be merged !

**Comment by aaillet** [comment]
Adding DNM, waiting for #60285 to be merged first (cc @xakep-amatop  )

**Review by xakep-amatop — CHANGES_REQUESTED** [review]

**Pull request closed without merging**
